test/v1/fixtures/RetrieveReliefInvestmentsFixtures.scala

```diff
@@ -42,21 +49,21 @@ object RetrieveReliefInvestmentsFixtures {
         """.stripMargin
   )
 
-  val eisSubscriptionsItemModel: EisSubscriptionsItem = EisSubscriptionsItem(
+  def eisSubscriptionsItemModel(knowledgeIntensive: Option[Boolean] = Some(true)): EisSubscriptionsItem = EisSubscriptionsItem(
     uniqueInvestmentRef = "XTAL",
     name = Some("EIS Fund X"),
-    knowledgeIntensive = true,
+    knowledgeIntensive = knowledgeIntensive,
     dateOfInvestment = Some("2020-12-12"),
     amountInvested = Some(BigDecimal(23312.00)),
     reliefClaimed = BigDecimal(43432.00)
   )
 
-  val eisSubscriptionsItemJson: JsValue = Json.parse(
-    """
+  def eisSubscriptionsItemJson(knowledgeIntensive: String = " | \"knowledgeIntensive\": true,"): JsValue = Json.parse(
```

Comment:
It might be confusing that callers are expected to pass a string that is a snippet of JSON and therefore must have some knowledge about the structure of the JSON. Given that this is the method that builds the JSON (and knows how to) it would be better to do it here and pass in an `Option[Boolean]` instead. You can implement either by piecing together the string internally as required or by adding an extra small JSON object  when required e.g.:
```
  def eisSubscriptionsItemJson(knowledgeIntensive: Option[Boolean] = Some(true)): JsValue =
    Json
      .parse(s"""
      |{
      |  "uniqueInvestmentRef": "XTAL",
      |  "name": "EIS Fund X",
      |  "dateOfInvestment": "2020-12-12",
      |  "amountInvested": 23312.00,
      |  "reliefClaimed": 43432.00
      |}""".stripMargin)
      .as[JsObject] ++
      knowledgeIntensive.map(x => Json.obj("knowledgeIntensive" -> x)).getOrElse(JsObject.empty)
```